tests/test_classifier.py

```diff
@@ -0,0 +1,103 @@
+from karton.classifier.classifier import Classifier
+import unittest
+from unittest.mock import ANY, MagicMock
+
+from karton.core import Resource, Task
+from karton.core.test import KartonTestCase, ConfigMock, KartonBackendMock
+from karton.classifier import Classifier
+
+
+class TestClassifier(KartonTestCase):
+
+    def setUp(self):
+        self.config = ConfigMock()
+        self.backend = KartonBackendMock()
+
+    def test_process(self):
```

Comment:
(copy&paste from my comment, because github won't allow me to request changes otherwise)

I feel like there's a lot of duplication, and I'm a bit concerned by the big number of new lines of code. For example, this fragment:

```python
m = MagicMock()
m.side_effect = [
    ', '.join([
        'PDF document',
        'version 1.4',
    ]),
    'application/pdf',
]
self.karton = Classifier(
    magic=m, config=self.config, backend=self.backend
)
```

with a good helper method could be shortened to:

```python
self.karton = mock_classifier(['PDF document', 'version 1.4'], "application/pdf")
```

And this fragment:

```python
resource = Resource('file.pdf', b'feeddecaf\n', sha256='sha256')
task = Task({
    'type': 'sample',
    'kind': 'raw',
})
task.add_payload('sample', resource)

res = self.run_task(task)
```

could be shortened to:

```python
res = classify("file.pdf")
```

this will add up over all the repeated invocations. Do you think it makes sense?

Also do you mind running `black` and `flake8` on your code? Looks like CI doesn't check `tests/` directory, but we like the consistency `black` enforces.